### PR TITLE
Remove CalendarToString and TimeZoneToString

### PR DESCRIPTION
**Pull request by ptomato**
See: #1294 (but does not close the discussion)

### CODE REVIEW

**Review by Ms2ger — APPROVED** [review]
Fair enough.

File: spec/calendar.html [hunk 1]
```diff
@@ -879,7 +867,7 @@ <h1>Temporal.Calendar.prototype.toJSON ( )</h1>
       </p>
       <emu-alg>
         1. Let _calendar_ be the *this* value.
-        1. Return ? CalendarToString(_calendar_).
+        1. Return ? ToString(_calendar_).
```
Comment (ljharb) [inline spec/calendar.html:870]: a reasonable alternative would be to Call %Temporal.Calendar.toString%, but this way plays better with subclassing i suppose.

**Review by ljharb — APPROVED** [review]

File: spec/calendar.html (hunk 1 above)
Comment (ptomato) [reply spec/calendar.html:870]: This way does an additional observable Get(calendar, @@toPrimitive) but I think that's OK?

Comment (ljharb) [reply spec/calendar.html:870]: Yes, either a Symbol.toPrimitive or a toString lookup would be required to invoke subclass code.

**Pull request merged**